sonata-project/admin-bundle/3.30/config/packages/sonata_admin.yaml

```diff
@@ -0,0 +1,2 @@
+sonata_admin:
+    title: 'Sonata Admin'
```

Comment:
Is this really needed? Do we need to add a config file?

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/DependencyInjection/Configuration.php#L82

ping @greg0ire

Reply:
Not needed no, but changing it is highly recommended, so…